tutorials/evoked/plot_eeg_erp.py

```diff
@@ -125,21 +125,23 @@
 
 ###############################################################################
 # Next, we create averages of stimulation-left vs stimulation-right trials.
-# We can use basic arithmetic to, for example, construct and plot
-# difference ERPs.
+# We can use negative weights in `mne.combine_evoked` to construct difference
+# ERPs.
```

Comment:
```suggestion
# of ERPs.
```

Reply:
I would say that "difference ERPs" is a valid term on its own, so no need to say "difference **of** ERPs" :)